src/Microsoft.VisualBasic/tests/Microsoft.VisualBasic.Tests.csproj

```diff
@@ -5,6 +5,7 @@
     <ProjectGuid>{82B54697-0251-47A1-8546-FC507D0F3B08}</ProjectGuid>
     <AssemblyName>Microsoft.VisualBasic.Tests</AssemblyName>
     <RootNamespace>Microsoft.VisualBasic.Tests</RootNamespace>
+    <UnsupportedPlatforms>Linux;NetBSD;OSX</UnsupportedPlatforms>
```

Comment:
So these will still need to be specified until the run-test.sh script is fixed. The run-test script will check and if there is a csproj file without a test dll then it will throw an error; however, if there is an UnsupportedPlatform tag then it will not error out (by design). The changes to fix run-test to work with missing binaries is out-of-scope for this change